pkg/codegen/jenny_schemareg.go

```diff
@@ -0,0 +1,88 @@
+package codegen
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing/fstest"
+
+	"github.com/grafana/codejen"
+	"github.com/grafana/grafana/pkg/cuectx"
+	"github.com/grafana/kindsys"
+	"github.com/grafana/thema"
+)
+
+// SchemaRegistryJenny generates lineage files into the "next" folder
+// of the local schema registry.
+func SchemaRegistryJenny(path string, latestRegistryDir string) OneToOne {
+	return &schemaregjenny{
+		path:              path,
+		latestRegistryDir: latestRegistryDir,
+	}
+}
+
+type schemaregjenny struct {
+	path              string
+	latestRegistryDir string
+}
+
+func (j *schemaregjenny) JennyName() string {
+	return "SchemaRegistryJenny"
+}
+
+func (j *schemaregjenny) Generate(kind kindsys.Kind) (*codejen.File, error) {
+	name := kind.Props().Common().MachineName
+	oldKindString, err := GetPublishedKind(name, "core", j.latestRegistryDir)
+	if err != nil {
+		return nil, err
+	}
+
+	var oldKind kindsys.Kind
+	if oldKindString != "" {
+		oldKind, err = loadCoreKind(name, oldKindString, "core")
```

Comment:
Nit; Having to pass the `"core"` category here, while the method is specifically named `loadCoreKind` looks suspicious. Indeed, seems that argument is not used within the `loadCoreKind` method.

Reply:
Nice catch, I fixed it in the new PR.